### DISCUSSION TITLE
generate_appcast throwing error 1001

### DISCUSSION THREAD

**Discussion by bvinicius** [sparkle-project/Sparkle#2253] [Q&A]
I've been using Sparkle for a while, but this week I tried to generate a new appcast.xml file as I usually do, but the command returns an error **nº 1001: No usable archives found in < path >.**

In my project root I have a directory called `Installer` where each target had its folder. So for the target `MyApp` this would be its app's path:

`./Installer/MyApp/Application/MyApp.app`

I also have a `sparkle` folder in the repository's root, containing the binaries commands, so in order to generate the Appcast I run the following:

`cd sparkle/`  
`./bin/generate_appcast ../Installer/MyApp/Application/`

But the command output is:

``` 
Error Domain=SUSparkleErrorDomain Code=1001 "No usable archives found in /Users/vbazanella/projects/mac/Installer/MyApp/Application" UserInfo={NSLocalizedDescription=No usable archives found in /Users/vbazanella/projects/mac/Installer/MyApp/Application}
```

I'm using Sparkle 2.0.  
I haven't made a recent update in Sparkle framework neither changed the contents inside `sparkle` directory.

Is there anything I'm missing?

Let me know if there's any additional information I need to give.

**Comment by zorgiepoo** [comment]
generate_appcast takes a directory that contains archives of your app you intend to upload/serve (not the actual .app's themselves).

Taken directly from its help page (on 2.x):

```
An example of an archives directory may look like:
    ./my-app-release-zipfiles/
        MyApp 1.0.zip
        MyApp 1.0.html
        MyApp 1.1.zip
        MyApp 1.1.html
        appcast.xml
```
